CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -245,6 +269,32 @@ function Assert-IsAgentServiceDisabled {
     throw "Agent service is not disabled. EXPECTED: Agent service is disabled"
 }
 
+function Test-IsAgentServiceRegistered {
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session)
+
+    $Service = Invoke-Command -Session $Session -ScriptBlock {
+        return $(if (Get-Service "ContrailAgent" -ErrorAction SilentlyContinue) { $true } else { $false })
+    }
+    return $Service
```

Comment:
These copy-pasted functions could get some more love. For example `return $()` is totally unnecessry here. Moreover, the `$Service` variable is also not necessary here. The whole function could be reduced to a single `Invoke-Command`.

(I know you just wanted to copy-paste, but I guess a little bit of clean up won't hurt :))

Reply:
Hmm, if I was going to refactor those functions, I would go all-out with the Pester's `Should`s etc. And we discussed that we should use asserts for now, until we migrate more tests.

Reply:
Agree on `Should`. But if we're leaving the `Test` version anyway, I'd still clean it up from unnecessary syntax noise:
```
Invoke-Command -Session $Session -ScriptBlock {
        if (Get-Service "ContrailAgent" -ErrorAction SilentlyContinue) { $true } else { $false }
}
```

Reply:
I will clean it up when I know how to make it work using Pester, e.g. I will probably rewrite the above line as something like

`Get-Service "ContrailAgent" | Should Not BeNull`

That's why I don't want to touch and clean up this code yet.